modules/renter/repair.go

```diff
@@ -150,6 +151,21 @@ func (f *file) chunkHosts(chunk uint64) []modules.NetAddress {
 	return old
 }
 
+// expiringContracts returns the contracts that will expire soon.
+// TODO: what if contract has fully expired?
```

Comment:
if the contract is already fully expired, then renew should not be attempted, instead it should be picked up by the repair loop.